src/pages/trader-tools/_tools.tsx

```diff
@@ -8,6 +8,11 @@ import { Header, Text, QueryImage } from 'components/elements'
 import { LinkButton } from 'components/form'
 import device from 'themes/device'
 
+type ColumnType = {
```

Comment:
This should be props.
```suggestion
type ColumnProps = {
```